test/jasmine/tests/parcoords_test.js

```diff
@@ -0,0 +1,1025 @@
+var Plotly = require('@lib/index');
+var Lib = require('@src/lib');
+var d3 = require('d3');
+var Plots = require('@src/plots/plots');
+var Parcoords = require('@src/traces/parcoords');
+var attributes = require('@src/traces/parcoords/attributes');
+
+var createGraphDiv = require('../assets/create_graph_div');
+var destroyGraphDiv = require('../assets/destroy_graph_div');
+var hasWebGLSupport = require('../assets/has_webgl_support');
+var mouseEvent = require('../assets/mouse_event');
+
+var mock0 = { // mock with zero dimensions; special case, as no dimension can be rendered
```

Comment:
> I tried it a while ago and it looked like there can't be a mock without a corresponding image baseline, otherwise the image test counts will mismatch and CI fails.

That's correct.

>  I believe you wanted to keep image tests low, for CI speed.

That's also correct. But, having 4 `parcoords` mocks sounds like a good compromise.

>  I'm glad to add image tests and png baselines for these, as I think it would be useful, please confirm though.

Yes. Go ahead and add the baselines. Thanks for double-checking with me :+1: